build.gradle

```diff
@@ -119,14 +119,10 @@ task stage {
 
 def environments = ['production', 'sandbox', 'alpha', 'crash']
 
-// TODO(mmuller): Move this into internal specialization code.
-def projects = ['production': 'domain-registry',
-                'sandbox'   : 'domain-registry-sandbox',
-                'alpha'     : 'domain-registry-alpha',
-                'crash'     : 'domain-registry-crash']
-
 def gcpProject = null
 
+apply from: "${rootDir.path}/projects.gradle"
```

Comment:
Sometimes I felt this kind of `apply` is quite invasive. We are just importing everything in the other file into this namespace, unqualified. Can we explicitly only import the `projects`?